python/tank/deploy/tank_commands/clone_configuration.py

```diff
@@ -232,7 +239,7 @@ def _do_clone(log, tk, source_pc_id, user_id, new_name, target_linux, target_mac
         
         # and write the new file
         fh = open(sg_pc_location, "wt")
-        yaml.dump(data, fh)
+        yaml.safe_dump(data, fh)
```

Comment:
It's not clear from yaml's documentation why safe_dump is better. What are we gaining? Any trade-offs?